### DISCUSSION TITLE
Configuring a maximum NAK PDU segment list size?

### DISCUSSION THREAD

**Discussion by jbell-astrobotic** [yamcs/yamcs#953] [General]
Poking around the Cfdp Service code + documentation and can't see any way to cap the segment list size for NAK PDUs. The size of this packet seems to be unbounded, and I would like to configure the maximum number of segments in a NAK to be in agreement with flight software and to ensure this packet can't exceed MTU limits. Tested with a very high packet loss environment and this NAK PDU seems to have the potential to grow quite large.

Currently on v5.9.2

**Comment by xpromache** [comment]
That issue was fixed back in April:
https://github.com/yamcs/yamcs/issues/830 

The limit is now the configured size of the PDU.

**Reply by jbell-astrobotic** [reply]
@xpromache awesome, thanks!
